numpy/_core/code_generators/generate_umath.py

```diff
@@ -1592,13 +1592,10 @@ def make_code(funcdict, filename):
     #include "matmul.h"
     #include "clip.h"
     #include "dtypemeta.h"
+    #include "dispatching.h"
     #include "_umath_doc_generated.h"
 
     %s
-    /* Returns a borrowed ref of the second value in the matching info tuple */
-    PyObject *
-    get_info_no_cast(PyUFuncObject *ufunc, PyArray_DTypeMeta *op_dtype,
-                     int ndtypes);
```

Comment:
not totally clear to me if originally not including this function in `dispatching.h` and forward-declaring it here was an intentional decision for some reason